content/articles/understanding-blockchain-as-a-service/index.md

```diff
@@ -0,0 +1,87 @@
+### Blockchain as a Service
+
+### Introduction
+The blockchain is the new buzzword in town and everyone seems to be talking about it. But how do businesses actually use the technology without having to build their own blockchains? Blockchain as a service (BaaS) enables companies to develop applications on various blockchains— such as Ethereum, Ripple or NEO —without having to code from scratch. Clients benefit from BaaS because it allows for faster application development, lower maintenance costs, and faster adoption of Blockchain technology.
+### Topic to handle
+- Defining Baas.
+- Why choose Baas.
+- What to check for while choosing Baas.
+- Providers of Baas.
+
+### Prerequisites
+To understand this article one should have:
+1. Good knowledge and understanding of how blockchain works.
+   
+### Definition of BaaS.
+Baas involves customers using cloud services to create, host, develop and run their blockchain applications, and smart contracts in the software and any other relevant operations that can run blockchain technology. The cloud services are offered and managed by the cloud services provider. They are also responsible for any other related task to keep the systems up and running.
+
+This is an introduction to BaaS, an important turning point in the blockchain ecosystem, with the potential to drive the widespread use of distributed ledger technologies industries. These are founded on and operates on principles that are alike to those used in SaaS. 
+
+###Why choose BaaS?
+Blockchain technology is increasingly being investigated by IT organizations across a wide range of industries for its strategic benefits. There are some technical difficulties, absence of domain expertise, and operational expenses associated with establishing, managing, and maintaining the Blockchain, as well as maintaining the infrastructure, typically lead key decision-makers to take their time with adoption plans.
```

Comment:
This talks about why companies have not adopted BaaS.

It would be great if you could talk about why companies should adopt BaaS? How's it different from other models?

Reply:
State any examples of companies using BaaS. How it operates?